src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/stats/FlowStatSpec.groovy

```diff
@@ -428,4 +430,52 @@ class FlowStatSpec extends HealthCheckSpecification {
         cleanup:
         flow && flowHelperV2.deleteFlow(flow.flowId)
     }
+
+    @Tidy
+    def "System is able to collect stats after partial updating(port) on a flow endpoint"() {
+        given: "Two active neighboring switches connected to the traffgens"
+        def traffGenSwitches = topology.activeTraffGens*.switchConnected*.dpId
+        def switchPair = topologyHelper.getAllNeighboringSwitchPairs().find {
```

Comment:
why only neighboring?

Reply:
No special reason, 
it was the first thing that occurred to me.
refactored.